<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>contacts</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color:rgb(68, 114, 196);</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>451</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Maiandra GD</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background:rgb(246,246,246)</string>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>79</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>firstName</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>lastName</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>phoneNumber</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Edit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Delete</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>161</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Maiandra GD</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Contact</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddNewContact">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Maiandra GD</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 28pt &quot;Maiandra GD&quot;;
 background-color: white;
 border-style: solid;
 border-width:1px;
 border-radius:25px;
 border-color: red;
 max-width:50px;
 max-height:50px;
 min-width:50px;
 min-height:50px;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearchContact">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Maiandra GD</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 28pt &quot;Maiandra GD&quot;;
 background-color: white;
 border-style: solid;
 border-width:1px;
 border-radius:12px;
 border-color: red;
 max-width:40px;
 max-height:40px;
 </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="firstNameInp">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="phoneNumberLbl">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Maiandra GD&quot;;</string>
   </property>
   <property name="text">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="FirstNameLbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Maiandra GD&quot;;</string>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="lastNameInp">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lastNameLbl">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Maiandra GD&quot;;</string>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="phoneNumberInp">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>440</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="nameToSearchInp">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>87</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdateContact">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>430</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Maiandra GD</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 28pt &quot;Maiandra GD&quot;;
 background-color: white;
 border-style: solid;
 border-width:1px;
 border-radius:12px;
 border-color: red;
 max-width:40px;
 max-height:40px;
 </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddContact">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>430</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Maiandra GD</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 28pt &quot;Maiandra GD&quot;;
 background-color: white;
 border-style: solid;
 border-width:1px;
 border-radius:12px;
 border-color: red;
 max-width:40px;
 max-height:40px;
 </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorLbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Maiandra GD&quot;;</string>
   </property>
   <property name="text">
    <string>Incorrect Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>contact_icon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
